### DISCUSSION TITLE
New installation - Login does not work

### DISCUSSION THREAD

**Discussion by w-wie-whisky** [alandtse/alexa_media_player#2530] [Q&A] [answered]
I tried to install the alexa media player.
HACS-Installation successfull, integration failed:

Integration asks for Amazon-Credentials (changed to amazon.de). Inserted my amazon-Username and Password - did not change anything else. New window opens asking me for Amazon Username, then password. Tried my amazon-Passwort - asks again for Username, then password. Tried my OTP - same loop - asks for Username and Password - and the loop is starting over and over...

HomeAssist:
Core 2024.9.1
Supervisor 2024.09.1
Operating System 13.1
Frontend 20240906.0

Alexa Media Player v 4.13.0

Anybody able to help?

**Comment by RandomUserName22** [comment]
Same thing is happening to me.  Did some housekeeping with my installation, had to remove and add Alexa, and can't authenticate.

**Comment by jarmenia** [comment]
I am seeing the same thing.  I am very new to HA so I assumed it was something I was doing.  Tried it both with my local URL as well as my HA Cloud URL.

**Comment by Nelapsi** [comment]
Same here 😄

**Comment by tebiwi** [comment, marked as answer]
Someone found this solution and it worked for me: 
After you've cycled through name, password and back to name, select the option Create Account New to Amazon?
Fill in a first & last name (anything at all as not going to matter so "Q W" is quick and easy to type!)
Amazon will then inform you that an account with your email already exists.
At that point re-select "Sign in Already a customer?" and proceed with the email and password windows and then the OTP window should appear. By that point though, the integration generated OTP has likely expired but you can just enter your current app generated OTP code and it should work.
If it returns to the integration add window and it just sits and spins and never progresses, that likely means the proxy listening session has expired and you'll have to repeat the process.

**Reply by Nelapsi** [reply]
Thank you very much !

**Reply by jarmenia** [reply]
Thank you, that worked for me as well!

**Reply by Paul-G-M** [reply]
Ahh, I can go to bed now... this work-around worked for me too. Thanks for sharing :)

**Comment by w-wie-whisky** [comment]
Thanks a lot! That worked for me.

**Comment by danielbrunt57** [comment]
Duplicate issue!  See issue #2514
